### PR TITLE
Bugfix/password reset

### PR DESCRIPTION
**Pull request by bleto**
# Description
- new infographic on authentication view,
- password recovery validation fix,

Issue #606 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have read the contribution requirements and fulfil them.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: modules/@ergonode/authentication/src/components/Forms/LoginCredentialsForm.vue [hunk 1]
```diff
@@ -129,7 +128,8 @@ export default {
 
 <style lang="scss" scoped>
     .header-logo {
-        justify-self: center;
+        display: block;
```
Comment (derpdead) [inline modules/@ergonode/authentication/src/components/Forms/LoginCredentialsForm.vue:131]: Default display might be already block

File: modules/@ergonode/authentication/src/components/Forms/NewPasswordForm.vue [hunk 2]
```diff
@@ -140,9 +128,29 @@ export default {
         ...mapActions('authentication', [
             'newPassword',
         ]),
+        validation(value, key) {
```
Comment (derpdead) [inline modules/@ergonode/authentication/src/components/Forms/NewPasswordForm.vue:131]: consider renaming to validatePassword

File: modules/@ergonode/authentication/src/components/Layout/Login.vue [hunk 3]
```diff
@@ -97,26 +112,32 @@ export default {
...
-            display: flex;
+            display: grid;
             flex: 1;
             justify-content: center;
-            align-items: center;
+            grid-auto-flow: column;
+            grid-template-columns: 1fr 1fr;
+            grid-column-gap: 16px;
+            width: 100%;
+            height: 100%;
```
Comment (derpdead) [inline modules/@ergonode/authentication/src/components/Layout/Login.vue:122]: height 100% is not recommended to set with flex-direction: column - already setting flex: 1 is expanding for full height

File: modules/@ergonode/authentication/src/components/Lists/ListElement.vue [hunk 4]
```diff
@@ -5,13 +5,13 @@
...
         <IconError
-            v-if="!item.value"
+            v-if="!item.success && !item.value"
             :fill-color="redColor"
             width="20"
             height="20" />
         <IconSuccess
             v-else
-            :fill-color="greenColor"
+            :fill-color="!item.value ? graphiteColor : greenColor"
```
Comment (derpdead) [inline modules/@ergonode/authentication/src/components/Lists/ListElement.vue:14]: consider migrating it into computed prop

File: modules/@ergonode/core/src/pages/login/index.vue [hunk 5]
```diff
@@ -5,14 +5,24 @@
...
-        <Component
-            :is="loginFormComponents.loginFormComponent"
-            @redirect-to="onRedirectTo" />
-        <Infographic :src="loginFormComponents.formImage" />
+        <template #form>
+            <Component
+                :is="loginFormComponents.formComponent"
+                @redirect-to="onRedirectTo" />
+        </template>
+        <FadeTransition>
```
Comment (derpdead) [inline modules/@ergonode/core/src/pages/login/index.vue:13]: FadeTransition wont apply - it only works for v-if / v-show

File: modules/@ergonode/core/src/pages/password-reset/index.vue [hunk 6]
```diff
@@ -4,8 +4,16 @@
...
 <template>
     <Login @redirect-to="onRedirectTo">
-        <NewPasswordForm />
-        <Infographic :src="require('@Core/assets/images/login/login_new_password_face.png')" />
+        <template #form>
+            <Component
+                :is="passwordResetFormComponents.formComponent"
+                @redirect-to="onRedirectTo" />
+        </template>
+        <FadeTransition>
```
Comment (derpdead) [inline modules/@ergonode/core/src/pages/password-reset/index.vue:12]: Same here, it will not work as intended

**Review by derpdead — CHANGES_REQUESTED** [review]

File: modules/@ergonode/core/src/pages/login/index.vue (hunk 5 above)
Comment (bleto) [reply modules/@ergonode/core/src/pages/login/index.vue:13]: It's an old code , I forgot to delete it

**Review by derpdead — APPROVED** [review]

**Pull request merged**